src/others/helpers/aid-request-helpers.test.ts

```diff
@@ -0,0 +1,125 @@
+import { groupByCities, groupByCategories, filterByCategoryIds } from "./aid-request-helpers";
+
+const exampleAggregatedRequests = [
+  { date: "2022-03-11", city_id: 1226, category_id: "sanitary_pads", requested_amount: 10 },
+  { date: "2022-03-12", city_id: 1226, category_id: "personal_hygiene_kits", requested_amount: 4 },
+  { date: "2022-03-11", city_id: 1, category_id: "bedding", requested_amount: 4 },
+  { date: "2022-03-11", city_id: 1226, category_id: "masks", requested_amount: 15 },
+  { date: "2022-03-10", city_id: 2, category_id: "personal_hygiene_kits", requested_amount: 1 },
+  { date: "2022-03-11", city_id: 1, category_id: "candles", requested_amount: 4 },
+  { date: "2022-03-11", city_id: 1226, category_id: "baby_products", requested_amount: 19 },
+  { date: "2022-03-11", city_id: 1226, category_id: "medical_kits_supplies", requested_amount: 16 },
+  { date: "2022-03-11", city_id: 1226, category_id: "torches", requested_amount: 10 },
+  { date: "2022-03-10", city_id: 1, category_id: "sanitary_pads", requested_amount: 2 },
+  { date: "2022-03-11", city_id: 1226, category_id: "water", requested_amount: 11 },
+  { date: "2022-03-10", city_id: 1, category_id: "food", requested_amount: 2 },
+  { date: "2022-03-11", city_id: 1226, category_id: "personal_hygiene_kits", requested_amount: 21 },
+  { date: "2022-03-11", city_id: 1, category_id: "batteries", requested_amount: 4 },
+  { date: "2022-03-11", city_id: 30, category_id: "batteries", requested_amount: 4 },
+  { date: "2022-03-10", city_id: 1, category_id: "baby_products", requested_amount: 2 },
+  { date: "2022-03-11", city_id: 1226, category_id: "tampons", requested_amount: 20 },
+  { date: "2022-03-11", city_id: 1226, category_id: "food", requested_amount: 10 },
+  { date: "2022-03-12", city_id: 1226, category_id: "medical_kits_supplies", requested_amount: 4 },
+  { date: "2022-03-12", city_id: 1226, category_id: "baby_products", requested_amount: 4 },
+  { date: "2022-03-11", city_id: 1226, category_id: "baby_food", requested_amount: 15 },
+];
+
+// TODO: pimp the tests
```

Comment:
what does this mean? 😅

Reply:
I probably meant "pimp up". And so I did!